src/main/java/io/jenkins/plugins/opentelemetry/opentelemetry/resources/JenkinsResourceProvider.java

```diff
@@ -23,7 +23,14 @@ public Resource createResource(ConfigProperties config) {
         ResourceBuilder resourceBuilder = Resource.builder();
         resourceBuilder.put(ResourceAttributes.SERVICE_NAME, JenkinsOtelSemanticAttributes.JENKINS);
         resourceBuilder.put(ResourceAttributes.SERVICE_NAMESPACE, JenkinsOtelSemanticAttributes.JENKINS);
-        // resourceBuilder.put(JenkinsOtelSemanticAttributes.JENKINS_URL, Jenkins.get().getRootUrl()); FIXME GET JENKINS URL
+
+        String jenkinsVersion = config.getString(JenkinsOtelSemanticAttributes.JENKINS_VERSION.getKey());
+        // Allow service.version to be overwritten and also report jenkins.version
+        resourceBuilder.put(ResourceAttributes.SERVICE_VERSION, jenkinsVersion);
+        resourceBuilder.put(JenkinsOtelSemanticAttributes.JENKINS_VERSION.getKey(), jenkinsVersion);
```

Comment:
https://github.com/jenkinsci/opentelemetry-plugin/pull/259 was the proposal to use the `SERVICE_VERSION`. What's the reason for this duplicated attribute?

Reply:
I duplicated because the `service.version` can be overwritten using environment variables /system properties. Maybe it's useless and we can remove `jenkins.version`

Reply:
Thanks Cyrille for the explanation!

Maybe it makes sense to keep it in case people wanna use the `service.version` to something else specific for Jenkins? Like the Helm chart (https://github.com/jenkinsci/helm-charts), for instance

```yaml
controller:
  componentName: "jenkins-controller"
  image: "jenkins/jenkins"
  tag: "2.303.3-jdk11"
  probes:
    livenessProbe:
      initialDelaySeconds: 240
      timeoutSeconds: 5
      failureThreshold: 12
    readinessProbe:
      initialDelaySeconds: 60
      periodSeconds: 10
  javaOpts: >-
      -Dotel.resource.attributes=service.version=3.10.2
```

O it does not make any sense?